src/xinterpreter.cpp

```diff
@@ -27,10 +27,7 @@
 using Args = std::vector<const char*>;
 
 void* createInterpreter(const Args &ExtraArgs = {}) {
-  Args ClangArgs = {/*"-xc++"*/"-v"}; // ? {"-Xclang", "-emit-llvm-only", "-Xclang", "-diagnostic-log-file", "-Xclang", "-", "-xc++"};
-#ifdef EMSCRIPTEN
-  ClangArgs.push_back("-std=c++20");
-#else
```

Comment:
We don't need to hardcode anything for xeus-cpp-lite from now on. The user can vary the kernel.json as per how he likes.